challenges/functions.js

```diff
@@ -6,25 +6,37 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+function consume(firstArg, secondArg, cb){
+   console.log(cb(firstArg, secondArg));
```

Comment:
**You need to return the callback** `function return the callback with the two parameters that you created`